internal/jobcontainers/jobcontainer.go

```diff
@@ -166,10 +201,29 @@ func (c *JobContainer) CreateProcess(ctx context.Context, config interface{}) (_
 		return nil, errors.New("unsupported process config passed in")
 	}
 
-	// Replace any occurrences of the sandbox mount point env variable in the commandline.
-	// %CONTAINER_SANDBOX_MOUNTPOINT%\mybinary.exe -> C:\C\123456789\mybinary.exe
+	// Replace any occurences of the sandbox mount env variable in the commandline.
+	// For example: %CONTAINER_SANDBOX_MOUNTPOINT%\mybinary.exe -> C:\<rootfslocation>\mybinary.exe.
 	commandLine, _ := c.replaceWithMountPoint(conf.CommandLine)
 
+	// This is to workaround a rather unfortunate outcome with launching a process in a silo that
+	// has bound files.
+	// If a user requested to launch a program at C:\<rootfslocation>\mybinary.exe because they
+	// expect C:\<rootfslocation>\mybinary.exe to exist once the file bindings are done, it actually
+	// won't work. This is because the executable is searched for using the parent processes filesystem view
+	// and not the containers/silos (that has access to these bound in files). Our Containerd shim is not
+	// one of these processes for example, and this is the parent process that is invoking CreateProcess so
```

Comment:
I still don't really understand what is being said here. What "processes" is our shim "not one of"?

Reply:
This can be improved a bit sorry. The processes being referred to are the silos processes where the outcome of not being assigned to the silo is we can't see any bound files. It's poorly worded/explained because the silo probably doesn't even have any processes running in it here. I'll clean this up

Reply:
Changed this around, please let me know if it's more clear